src/controllers/v1/resource.js

```diff
@@ -81,4 +81,32 @@ module.exports = class Resource {
 			throw error
 		}
 	}
+
+	/**
+	 * Get list of Published resources from consumption side
+	 * @method
+	 * @name browseExisting
+	 * @param {String} req.decodedToken.organization_id - User organization id.
+	 * @param {String} req.decodedToken.token - user bearer auth token
+	 * @param {Object} req.query - Query object passed
+	 * @param {String} req.searchText - Search text passed
+	 * @param {Integer} pageNo -  Used to skip to different pages. Used for pagination . If value is not passed, by default it will be 1
+	 * @param {Integer} pageSize -  Used to limit the data. Used for pagination . If value is not passed, by default it will be 100
+	 * @returns {JSON} - List of resources
+	 */
+	async browseExisting(req) {
```

Comment:
@adithyadinesh0412 add this in validator and make sure if the type which frontend is passing valid